src/tests/ftest/osa/osa_offline_parallel_test.py

```diff
@@ -79,27 +99,41 @@ def run_offline_parallel_test(self, num_pool, data=False):
         # Exclude rank 2.
         rank = 2
 
+        test_seq = self.ior_test_sequence[0]
         for val in range(0, num_pool):
             pool[val] = TestPool(self.context,
                                  dmg_command=self.get_dmg_command())
             pool[val].get_params(self)
-            # Split total SCM and NVME size for creating multiple pools.
-            pool[val].scm_size.value = int(pool[val].scm_size.value /
-                                           num_pool)
-            pool[val].nvme_size.value = int(pool[val].nvme_size.value /
-                                            num_pool)
             pool[val].create()
             pool_uuid.append(pool[val].uuid)
             self.pool = pool[val]
             if data:
-                self.write_single_object()
+                self.run_ior_thread("Write", oclass, test_seq)
+                if oclass != "S1":
+                    self.run_mdtest_thread()
+                if self.test_during_aggregation is True:
+                    self.run_ior_thread("Write", oclass, test_seq)
+
+        # Start the additional servers and extend the pool
+        self.log.info("Extra Servers = %s", self.extra_servers)
+        self.start_additional_servers(self.extra_servers)
+        # Give sometime for the additional server to come up.
+        for retry in range(0, 10):
+            scan_info = self.get_dmg_command().system_query()
+            if not check_system_query_status(scan_info):
+                if retry == 9:
+                    self.fail("One or more servers not in expected status")
+            else:
+                break
 
         # Exclude and reintegrate the pool_uuid, rank and targets
         for val in range(0, num_pool):
             self.pool = pool[val]
             self.pool.display_pool_daos_space("Pool space: Beginning")
             pver_begin = self.get_pool_version()
             self.log.info("Pool Version at the beginning %s", pver_begin)
+            if self.test_during_aggregation is True and val == 0:
+                self.delete_extra_container(self.pool)
```

Comment:
Why do you delete extra container only when val == 0?

Reply:
You are correct... I think there is no restriction to it only for val=0... Will remove it... Unless, there is an issue for running a specific test case.